packages/common-substrate/src/project/versioned/v1_0_0/model.ts

```diff
@@ -147,10 +133,18 @@ export class DeploymentV1_0_0 {
   parent?: ParentProject;
 }
 
-export class ProjectManifestV1_0_0Impl<D extends object = DeploymentV1_0_0>
-  extends ProjectManifestBaseImpl<D>
+export class ProjectManifestV1_0_0Impl
+  extends ProjectManifestBaseImpl<DeploymentV1_0_0>
   implements SubstrateProjectManifestV1_0_0
 {
+  constructor() {
+    super(DeploymentV1_0_0);
+  }
+
+  toYaml(): string {
```

Comment:
I don't think this is used. Its also a bit confusing because it gives you a deployment yaml not a project yaml